invenio/templates/application.yaml

```diff
@@ -38,6 +38,16 @@ spec:
       labels:
         app: web
     spec:
+      initContainers:
```

Comment:
I was thinking that maybe the initContainer  could be used to initialize the NGINX Pod instead since there will be always fewer Nginx-Pods and maybe moving it there, you could remove the need for using a persistent volume and just copy it to a local volume of this pod(unsure if you're going to have the same mounting-wiping issue.

Reply:
Discussed IRL, to be seen after testing nginx as sidecar.